test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/SelectTagHelperTest.cs

```diff
@@ -356,6 +356,91 @@ public class SelectTagHelperTest
             Assert.Equal(savedValue, items.Select(item => item.Value));
         }
 
+        [Fact]
+        public async Task ProcessAsync_WithItems_AndNoModelExpression_GeneratesExpectedOutput()
+        {
+            // Arrange
+            var originalAttributes = new TagHelperAttributeList
+            {
+                { "class", "form-control" },
+            };
+            var originalPostContent = "original content";
+
+            var expectedAttributes = new TagHelperAttributeList(originalAttributes);
+            var selectItems = new SelectList(Enumerable.Range(0, 5));
+            var expectedOptions = string.Join(
+                Environment.NewLine,
+                selectItems.Select(i => $"<option>HtmlEncode[[{i.Text}]]</option>")) + Environment.NewLine;
```

Comment:
Correct to use `Environment.NewLine` but don't make anyone guess what is expected. Show the HTML, not an expression to generate it.